kong/plugins/jwt/api.lua

```diff
@@ -53,5 +53,34 @@ return {
     DELETE = function(self, dao_factory)
       crud.delete(self.jwt_secret, dao_factory.jwt_secrets)
     end
+  },
+  ["/jwts/"] = {
+    GET = function(self, dao_factory)
+      crud.paginated_set(self, dao_factory.jwt_secrets)
+    end
+  },
+  ["/jwts/:credential_key_or_id/consumer"] = {
```

Comment:
Would `jwt_key_or_id` be a better name here? The JWT and it being a credential from the point of view of the Kong data model might not be obvious enough at first sight.

Reply:
Makes sense, will push it right away!

Reply:
@thibaultcha Line 23 in the same file. Should I change those to keep it consistent?